### PR TITLE
Updating draft of 21.0.0.4 GA 

### PR DESCRIPTION
**Pull request by jakub-pomykala**

### CODE REVIEW

File: posts/2021-04-16-mp-context-propagration-21004.adoc [hunk 1]
```diff
@@ -123,7 +123,7 @@ Stay tuned for a blog specifically regarding Admin Center which will showcase th
...
+LDAP bind operations are used to authenticate clients (and the users or applications behind them) to the directory server. This establishes an authorization identity that is used for subsequent operations that are processed on that connection, and specifies the LDAP protocol version that the client uses. Before this update, the `LdapRegistry` element supported binding either anonymously or by using simple authentication with a user (`bindDN`) and password (`bindPassword`). This update adds an option to bind to LDAP: GSSAPI/Kerberos. Kerberos is an authentication mechanism that allows a client to authenticate by a Key Distribution Center (KDC).. In Open Liberty 21.0.0.4-beta,  you can use either a Kerberos credential cache (ccache) or a Kerberos keytab file.
```
Comment (austin0) [inline posts/2021-04-16-mp-context-propagration-21004.adoc]:
```suggestion
LDAP bind operations are used to authenticate clients (and the users or applications behind them) to the directory server. This establishes an authorization identity that is used for subsequent operations that are processed on that connection, and specifies the LDAP protocol version that the client uses. Before this update, the `LdapRegistry` element supported binding either anonymously or by using simple authentication with a user (`bindDN`) and password (`bindPassword`). This update adds an option to bind to LDAP: GSSAPI/Kerberos. Kerberos is an authentication mechanism that allows a client to authenticate by a Key Distribution Center (KDC). In Open Liberty 21.0.0.4-beta,  you can use either a Kerberos credential cache (ccache) or a Kerberos keytab file.
```

File: posts/2021-04-16-mp-context-propagration-21004.adoc [hunk 2]
```diff
@@ -139,41 +139,41 @@ You also need the Kerberos principal or Service Principal Name:
...
+Set the Kerberos configuration file name (for example, krb5.conf or krb5.ini) using the Kerberos element:
 
 [source, xml]
 ----
 <kerberos configFile="${server.config.dir}/security/krb5.conf"/>
 ----
-
-If you are using a Kerberos keytab file to store encrypted keys for principals, set the file using the global Kerberos configuration element:
+s
+If you are using a Kerberos keytab file, set the Kerberos keytab file name using the Kerberos element:
```
Comment (austin0) [inline posts/2021-04-16-mp-context-propagration-21004.adoc]:
```suggestion

If you are using a Kerberos keytab file, set the Kerberos keytab file name using the Kerberos element:
```

**Review by austin0 — CHANGES_REQUESTED** [review]

File: posts/2021-04-16-mp-context-propagration-21004.adoc (hunk 1 above)
Comment (jakub-pomykala) [reply posts/2021-04-16-mp-context-propagration-21004.adoc]: Done

File: posts/2021-04-16-mp-context-propagration-21004.adoc (hunk 2 above)
Comment (jakub-pomykala) [reply posts/2021-04-16-mp-context-propagration-21004.adoc]: Done

**Review by austin0 — APPROVED** [review]

**Pull request merged**